shaders/pbr-frag.glsl

```diff
@@ -254,7 +254,7 @@ void main()
     vec3 reflection = -normalize(reflect(v, n));
 
     float NdotL = clamp(dot(n, l), 0.001, 1.0);
-    float NdotV = abs(dot(n, v)) + 0.001;
+    float NdotV = min(abs(dot(n, v)) + 0.001, 1.0);
```

Comment:
Any reason to not use `clamp` here, just to be consistent with the surrounding code?

Reply:
To preserve existing behavior and keep the changes to a minimum.
Should it be `clamp(abs(dot(n, v))+0.001, 0.0, 1.0);` or `clamp(abs(dot(n, v)), 0.001, 1.0);`?

Reply:
Looks like the `0.001` is there to prevent a divide-by-zero, so yes I would think that `clamp(abs(dot(n, v)), 0.001, 1.0);` is the correct one.